dashboard/app/jobs_test.go

```diff
@@ -113,9 +114,6 @@ func TestJob(t *testing.T) {
 			"repro syz"))
 	c.expectEQ(pollResp.ReproC, []byte("repro C"))
 
-	pollResp2 := client.pollJobs(build.Manager)
```

Comment:
Could you please help me understand why it was removed?
It looks like we're changing API idempotency if this line started to fail.

Reply:
Earlier the API would return the same job every poll until the job was completed. This PR changes this behavior, now the job is returned only once (until `job_reset` is called), that's why the line was deleted.

If a syz-ci is newer than dashboard, syz-ci just won't do any jobs (reset would fail and the job loop wouldn't start).
A slightly outdated syz-ci should work fine with a new dashboard for some time (then failed jobs could accumulate), but I don't think there are any promises that this combination should work in the long term.